src/entities/tokens.ts

```diff
@@ -110,3 +110,30 @@ export interface RainbowToken extends Asset {
 export interface IndexToken extends Asset {
   amount: string;
 }
+
+export interface ExchangeAsset {
```

Comment:
considering we have a `SwappableAsset` type as well I wonder if we're able to combine the types (havnt looked at the diff yet)